doc/release/README.chplenv

```diff
@@ -351,9 +351,11 @@ Optional Settings
 
 *  Optionally, the CHPL_HWLOC environment variable can select between
    no hwloc support or using the hwloc package distributed with Chapel
-   in third-party.  Note that hwloc is only used by the qthreads
-   tasking layer, and does not need to be built for other tasking
-   layers.  Current options are:
+   in third-party (see www.open-mpi.org/projects/hwloc/ for more information 
```

Comment:
I think this is acceptable, yet the URL appears in third-party/README and in third-party/hwloc/README, and I think the fewer places we have it, this less maintenance is retired if/when the package moves.  I'd either refer to one of the above READMEs here, or figure that people will be able to find it on their own given the textual description of where it lives.